src/backend/src/transformers/change-requests.transformer.ts

```diff
@@ -9,7 +15,7 @@ import userTransformer from './user.transformer';
 const changeRequestTransformer = (
   changeRequest: Prisma.Change_RequestGetPayload<typeof changeRequestRelationArgs>
 ): ChangeRequest | StandardChangeRequest | ActivationChangeRequest | StageGateChangeRequest => {
-  return {
+  const cr = {
```

Comment:
with my other comment, we wont need this weirdness and you'll just be able to add
`status: calculateChangeRequestStatus(changeRequest)`